vivarium/core/store.py

```diff
@@ -1006,6 +1006,29 @@ def emit_data(self):
             return self.value
         return None
 
+    def toggle_emits(self, emit=False, paths=None):
```

Comment:
To me, "toggle" means to convert on to off and off to on. Instead, this function sets emits to be on/off. For example if you call `toggle_emits(emit=True, paths=[('a',)])` and `('a',)` already has its emit enabled, this function doesn't toggle the emit to off.

Reply:
I renamed these methods `set_emit_values` and `set_emit_value`